judge/utils/problems.py

```diff
@@ -80,7 +80,7 @@ def get_result_table(*args, **kwargs):
             (_('Compile Error'), 'CE', results['CE']),
             (_('Time Limit Exceeded'), 'TLE', results['TLE']),
             (_('Memory Limit Exceeded'), 'MLE', results['MLE']),
-            (_('Invalid Return'), 'IR', results['IR']),
```

Comment:
Unless I'm wrong, that `'IR'` is used to determine a CSS class for coloring the table. You'll need to change that CSS class to "O".